### PR TITLE
Release NCHS Mortality

### PR DESCRIPTION
**Pull request by krivard**
Adds nchs-mortality.md.

This is the same as the text from [this mysteriously closed PR](https://github.com/cmu-delphi/delphi-epidata/pull/264) (past-krivard, what were you doing?), plus first issue and data licensing information.

### CODE REVIEW

File: docs/api/covidcast-signals/nchs-mortality.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+certificate data received and coded by the National Center for Health Statistics
+[(NCHS)](https://www.cdc.gov/nchs/nvss/vsrr/COVID19/index.htm).
+
+| Signal | Description |
+| --- | --- |
+| `deaths_covid_incidence_num` | Number of weekly new deaths with confirmed or presumed COVID-19 |
+| `deaths_covid_incidence_prop` | Number of weekly new deaths with confirmed or presumed COVID-19, per 100,000 population |
+| `deaths_allcause_incidence_num` | Number of weekly new deaths from all causes |
+| `deaths_allcause_incidence_prop` | Number of weekly new deaths from all causes, per 100,000 population |
+| `deaths_flu_incidence_num` | Number of weekly new deaths involving Influenza |
```
Comment (jingjtang) [inline docs/api/covidcast-signals/nchs-mortality.md:27]: Number of weekly new deaths involving Influenza and pneumonia or deaths involving influenza and COVID-19

File: docs/api/covidcast-signals/nchs-mortality.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+| `deaths_flu_incidence_prop` | Number of weekly new deaths involving Influenza, per 100,000 population |
```
Comment (jingjtang) [inline docs/api/covidcast-signals/nchs-mortality.md:28]: Number of weekly new deaths involving Influenza and pneumonia or deaths involving influenza and COVID-19, per 100, 000 population

File: docs/api/covidcast-signals/nchs-mortality.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+## Lag and Backfill
+
+There is a lag in time between when the death occurred and when the death
+certificate is completed, submitted to NCHS, and processed for reporting
+purposes. The death counts for earlier weeks are continually revised and may
+increase or decrease as new and updated death certificate data are received from
+the states by NCHS. This delay can range from 1 week to 8 weeks or even more. We
+check for updates reported by NCHS every weekday but will report the signals
+weekly (on Monday). The data changes due to backfill made over the past week can
+be fetched from our API.
```
Comment (jingjtang) [inline docs/api/covidcast-signals/nchs-mortality.md:79]: We can add these two lines or not. I borrowed them from CDC website.

This delay can range from 1 week to 8 weeks or even more. Some states report deaths on a daily basis, while other states report deaths weekly or monthly. State vital record reporting may also be affected or delayed by COVID-19 related response activities which makes death counts not comparable across states. We check for updates reported by NCHS every weekday but will report the signals weekly (on Monday).

**Comment by jingjtang** [comment]
About missingness, according to NCHS, 
Data for jurisdictions where counts are between 1 and 9 are suppressed. Additionally, data for weeks where the counts are less than 50% of the expected number are also suppressed, as these provisional counts are highly incomplete and potentially misleading.

File: docs/api/covidcast-signals/nchs-mortality.md (hunk 1 above)
Comment (krivard) [reply docs/api/covidcast-signals/nchs-mortality.md:27]: If I enumerate all possibilities, which are included in `deaths_flu_incidence_num`?

| | Influenza | Pneumonia | COVID-19
| - | ---------- | ------------ | -----------
1 | yes | no | no
2 | yes | yes | no
3 | yes | no | yes
4 | yes | yes | yes

File: docs/api/covidcast-signals/nchs-mortality.md (hunk 3 above)
Comment (krivard) [inline docs/api/covidcast-signals/nchs-mortality.md:79]:
```suggestion
the states by NCHS. This delay can range from 1 week to 8 weeks or even more.
Some states report deaths on a daily basis, while other states report deaths weekly
or monthly. State vital record reporting may also be affected or delayed by
COVID-19 related response activities which makes death counts not comparable
across states. We check for updates reported by NCHS every weekday but will
report the signals weekly (on Monday).
```

File: docs/api/covidcast-signals/nchs-mortality.md (hunk 1 above)
Comment (jingjtang) [reply docs/api/covidcast-signals/nchs-mortality.md:27]: 2，3，4 will be included

Comment (krivard) [reply docs/api/covidcast-signals/nchs-mortality.md:27]:
```suggestion
| `deaths_flu_incidence_num` | Number of weekly new deaths involving Influenza and at least one of (Pneumonia, COVID-19)|
```

File: docs/api/covidcast-signals/nchs-mortality.md (hunk 2 above)
Comment (krivard) [reply docs/api/covidcast-signals/nchs-mortality.md:28]:
```suggestion
| `deaths_flu_incidence_prop` | Number of weekly new deaths involving Influenza and at least one of (Pneumonia, COVID-19), per 100,000 population |
```

**Review by krivard — COMMENTED** [review]
Wording fixes for flu signals

File: docs/api/covidcast-signals/nchs-mortality.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+| `deaths_flu_incidence_prop` | Number of weekly new deaths involving Influenza and at least one of (Pneumonia, COVID-19), per 100,000 population |
+| `deaths_pneumonia_notflu_incidence_num` | Number of weekly new deaths involving Pneumonia, excluding Influenza deaths |
+| `deaths_pneumonia_notflu_incidence_prop` | Number of weekly new deaths involving Pneumonia, excluding Influenza deaths, per 100,000 population |
+| `deaths_covid_and_pneumonia_notflu_incidence_num`| Number of weekly new deaths involving COVID-19 and Pneumonia, excluding Influenza |
+| `deaths_covid_and_pneumonia_notflu_incidence_prop`| Number of weekly new deaths involving COVID-19 and Pneumonia, excluding Influenza, per 100,000 population |
+|`deaths_pneumonia_or_flu_or_covid_incidence_num`| Number of weekly new deaths involving Pneumonia, Influenza, or COVID-19|
+|`deaths_pneumonia_or_flu_or_covid_incidence_prop`| Number of weekly new deaths involving Pneumonia, Influenza, or COVID-19, per 100,000 population|
+|`deaths_percent_of_expected`| The number of weekly new deaths for all causes in 2020 compared to the average number across the same week in 2017–2019|
```
Comment (Akvannortwick) [inline docs/api/covidcast-signals/nchs-mortality.md:35]:
```suggestion
|`deaths_percent_of_expected`| Number of weekly new deaths for all causes in 2020 compared to the average number across the same week in 2017–2019|
```

If we are being consistent with the rest of the wording above.

File: docs/api/covidcast-signals/nchs-mortality.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+
+## Table of contents
+{: .no_toc .text-delta}
+
+1. TOC
+{:toc}
+
+## Geographical Exceptions
+
+New York City is considered as a special state in the NCHS Mortality data, but
```
Comment (Akvannortwick) [inline docs/api/covidcast-signals/nchs-mortality.md:56]:
```suggestion
New York City is considered a special state in the NCHS Mortality data, but
```

File: docs/api/covidcast-signals/nchs-mortality.md [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+expected number, since these provisional counts are highly incomplete and
+potentially misleading.
+
+## Lag and Backfill
+
+There is a lag in time between when the death occurred and when the death
+certificate is completed, submitted to NCHS, and processed for reporting
+purposes. The death counts for earlier weeks are continually revised and may
+increase or decrease as new and updated death certificate data are received from
+the states by NCHS. This delay can range from 1 week to 8 weeks or even more.
```
Comment (Akvannortwick) [inline docs/api/covidcast-signals/nchs-mortality.md:84]:
```suggestion
the states by NCHS. This delay can range from 1 to 8 weeks or even more.
```

File: docs/api/covidcast-signals/nchs-mortality.md (hunk 5 above)
Comment (krivard) [reply docs/api/covidcast-signals/nchs-mortality.md:56]:
```suggestion
New York City is listed as its own region in the NCHS Mortality data, but
```

**Comment by krivard** [comment]
We'll merge this Thursday morning after checking metadata.

**Pull request merged**